New Script: [Memory] Copy and Paste Viewport Placement

gtalarico opened the pull request:
@eirannejad this is the copy/paste view port placement script I mentioned to you.
feel free to modify as you wish, including header/notice, etc.

eirannejad commented:
Gui,Thanks so much for these. I added them to the repo and also added the
Author to the tooltips. I made major modifications to these so these tools
can convert coordinates between model and sheet space and position the
viewports based on the model space on the sheets. This is very important
for architects to have all the plan viewports aligned on multiple sheets
for coordination and overlay.

So I added my name to the author after yours. Let me know if this is okay
with u :)

Thanks a bunch again!

On 10 August 2016 at 15:51, Gui Talarico <notifications@github.com> wrote:

> @eirannejad <https://github.com/eirannejad> this is the copy/paste view
> port placement script I mentioned to you.
> feel free to modify as you wish, including header/notice, etc.
> ------------------------------
> You can view, comment on, or merge this pull request online at:
>
>   https://github.com/eirannejad/pyRevit/pull/118
> Commit Summary
>
>    - New Script: Copy and Paste Viewport Placement
>
> File Changes
>
>    - *A* pyRevit/Memory_copyViewPortPlacement.png
>    <https://github.com/eirannejad/pyRevit/pull/118/files#diff-0> (0)
>    - *A* pyRevit/Memory_copyViewPortPlacement.py
>    <https://github.com/eirannejad/pyRevit/pull/118/files#diff-1> (47)
>    - *A* pyRevit/Memory_pasteViewPortPlacement.png
>    <https://github.com/eirannejad/pyRevit/pull/118/files#diff-2> (0)
>    - *A* pyRevit/Memory_pasteViewPortPlacement.py
>    <https://github.com/eirannejad/pyRevit/pull/118/files#diff-3> (56)
>
> Patch Links:
>
>    - https://github.com/eirannejad/pyRevit/pull/118.patch
>    - https://github.com/eirannejad/pyRevit/pull/118.diff
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/eirannejad/pyRevit/pull/118>, or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AH0XHMXOuDynLJGLFLT90UCVREYQKVcJks5qelXcgaJpZM4Jhp_U>
> .
>

gtalarico commented:
Makes perfect sense.
Thanks for improving it!

Worked for me because I had all views cropped to the same scope box, so it
was kind of "hacky" :)

Gui Talarico
[ mobile ]

On Aug 12, 2016 12:38 PM, "Ehsan Iran-Nejad" <notifications@github.com>
wrote:

> Gui,Thanks so much for these. I added them to the repo and also added the
> Author to the tooltips. I made major modifications to these so these tools
> can convert coordinates between model and sheet space and position the
> viewports based on the model space on the sheets. This is very important
> for architects to have all the plan viewports aligned on multiple sheets
> for coordination and overlay.
>
> So I added my name to the author after yours. Let me know if this is okay
> with u :)
>
> Thanks a bunch again!
>
> On 10 August 2016 at 15:51, Gui Talarico <notifications@github.com> wrote:
>
> > @eirannejad <https://github.com/eirannejad> this is the copy/paste view
> > port placement script I mentioned to you.
> > feel free to modify as you wish, including header/notice, etc.
> > ------------------------------
> > You can view, comment on, or merge this pull request online at:
> >
> > https://github.com/eirannejad/pyRevit/pull/118
> > Commit Summary
> >
> > - New Script: Copy and Paste Viewport Placement
> >
> > File Changes
> >
> > - *A* pyRevit/Memory_copyViewPortPlacement.png
> > <https://github.com/eirannejad/pyRevit/pull/118/files#diff-0> (0)
> > - *A* pyRevit/Memory_copyViewPortPlacement.py
> > <https://github.com/eirannejad/pyRevit/pull/118/files#diff-1> (47)
> > - *A* pyRevit/Memory_pasteViewPortPlacement.png
> > <https://github.com/eirannejad/pyRevit/pull/118/files#diff-2> (0)
> > - *A* pyRevit/Memory_pasteViewPortPlacement.py
> > <https://github.com/eirannejad/pyRevit/pull/118/files#diff-3> (56)
> >
> > Patch Links:
> >
> > - https://github.com/eirannejad/pyRevit/pull/118.patch
> > - https://github.com/eirannejad/pyRevit/pull/118.diff
> >
> > —
> > You are receiving this because you were mentioned.
> > Reply to this email directly, view it on GitHub
> > <https://github.com/eirannejad/pyRevit/pull/118>, or mute the thread
> > <https://github.com/notifications/unsubscribe-auth/
> AH0XHMXOuDynLJGLFLT90UCVREYQKVcJks5qelXcgaJpZM4Jhp_U>
> > .
> >
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/eirannejad/pyRevit/pull/118#issuecomment-239495777>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AJEr8H2k73X9FZ3l5roDpfLfvyQwz-08ks5qfKFqgaJpZM4Jhp_U>
> .
>
